jsk_pr2_robot/jsk_pr2_startup/pr2_gazebo.launch

```diff
@@ -113,11 +113,17 @@
     <node name="relay_c2_depth_registered_camera_info"
           pkg="topic_tools" type="relay"
           args="/head_mount_kinect/depth_registered/camera_info /kinect_head_c2/depth_registered/camera_info" />
+    <node name="kinect_c2_depth_registered_points_throttle"
+          pkg="topic_tools" type="throttle"
+          args="messages /kinect_head_c2/depth_registered/points 4.0 /kinect_head_c2/depth_registered/points_throttle" />
   </group>
 
   <node name="relay_cmd_vel"
         pkg="topic_tools" type="relay"
         args="/navigation/cmd_vel /navigation/unsafe_vel" />
+  <!-- Set params for sim -->
+  <param name="r_gripper_controller/gripper_action_node/stall_velocity_threshold" value="0.33" type="double"/>
```

Comment:
This change should be reverted after release of https://github.com/PR2/pr2_simulator/pull/115